src/BloomBrowserUI/bookEdit/bloomField/bloomField.ts

```diff
@@ -0,0 +1,254 @@
+/// <reference path="../../lib/jquery.d.ts" />
+
+// We have a lot of trouble with the embedded Firefox's ContentEditable behavior. The chief problem is that FF *really* wants to just use <br>s,
+// which are worthless because you can't style them to, for example, do paragraph indents.
+// So this module is for various work-arounds. Eventually we should probably admit defeat and move to some existing open-source div editor.
+
+//Firefox adds a <BR> when you press return, which is lame because you can't put css styles on BR, such as indent.
+//Eventually we may use a wysiwyg add-on which does this conversion as you type, but for now, we change it when
+//you tab or click out.
+
+enum CursorPosition {start, end}
+
+class BloomField {
```

Comment:
I see six lines of comment there. Now, it might not answer your question, so let's see what we can do about that. Note, this isn't a class in the sense you're thinking... there is no state. These are all static methods, and they all  take one parameter, the field. Now, what I would like to do is control the visibility, since all but the first method should be private but I couldn't get that to work. I could try again.

>I'm not sure for example whether it is ONE editable field 

Hmm. Woulda thought that the variable name "field" doesn't conveyed that "one".